tests/test_functional.py

```diff
@@ -33,59 +32,26 @@
 
 from pylint import testutils
 from pylint.testutils import UPDATE_FILE, UPDATE_OPTION
-from pylint.testutils.functional_test_file import FunctionalTestFile
+from pylint.testutils.functional import (
+    FunctionalTestFile,
+    get_functional_test_files_from_directory,
+)
+from pylint.testutils.functional.lint_module_output_update import LintModuleOutputUpdate
```

Comment:
Isn't the idea that we should now import this from `pylint.testutils.functional`?

Reply:
Good catch, I messed up because inside pylint.testutil you need the fullpath to not create circular import.

Reply:
5088b22